azurerm/internal/services/cognitive/resource_arm_cognitive_account.go

```diff
@@ -98,6 +98,32 @@ func resourceArmCognitiveAccount() *schema.Resource {
 				}, false),
 			},
 
+			"properties": {
```

Comment:
Could we remove this block and pull its properties out into the top level?

Reply:
the api_properties block has now been moved to the root